examples/example_0008_modify_turbine.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2019 NREL
+
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+# this file except in compliance with the License. You may obtain a copy of the
+# License at http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+# See read the https://floris.readthedocs.io for documentation
+
+# This example illustrates changing the properties of some of the turbines
+# This can be used to setup farms of different turbines
+
+import matplotlib.pyplot as plt
+import floris.tools as wfct
+
+# Initialize the FLORIS interface fi
+fi = wfct.floris_interface.FlorisInterface("example_input.json")
+
+# Change turbine 0, 3 to have a 35 m rotor diamater
```

Comment:
The comment could be clearer that you are changing turbines 0 and 3:

\# Change turbine 0 and 3 to have a 35 m rotor diameter